templates/original-permalink-template.php

```diff
@@ -38,11 +38,14 @@
 										<summary>All Languages</summary>
 										<?php if ( $translations_by_locale ) : ?>
 											<ul class="suggestions-list">
-											<?php foreach ( $translations_by_locale as $locale => $translation ) : ?>
+											<?php
+											foreach ( $translations_by_locale as $locale => $translation ) : // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+												?>
+
 												<li>
 													<div class="translation-suggestion with-tooltip" tabindex="0" role="button" aria-pressed="false" aria-label="Copy translation">
 														<span class="translation-suggestion__translation">
-															<?php echo strtoupper( $locale ) . ' - ' . esc_html( $translation ); ?>
+															<?php echo esc_html( strtoupper( $locale ) ) . ' - ' . esc_html( $translation ); ?>
```

Comment:
```suggestion
															<?php echo esc_html( strtoupper( $_locale ) ) . ' - ' . esc_html( $translation ); ?>
```

Reply:
It looks fine to me with the previous suggestion (see previous commit).